src/common/RunParams.cpp

```diff
@@ -411,17 +411,17 @@ void RunParams::printHelpMessage(std::ostream& str) const
       << "\t      (print names of features used by each kernel)\n\n";
 
   str << "\t --npasses <int> [default is 1]\n"
-      << "\t      (num passes through suite)\n"; 
+      << "\t      (num passes through Suite)\n"; 
   str << "\t\t Example...\n"
-      << "\t\t --npasses 2 (runs complete suite twice\n\n";
+      << "\t\t --npasses 2 (runs complete Suite twice\n\n";
 
   str << "\t --repfact <double> [default is 1.0]\n"
-      << "\t      (fraction of default # reps to run each kernel)\n";
+      << "\t      (percentage of default # reps to run each kernel)\n";
```

Comment:
I'm not sure percentage is the right way to say this. This would seem to say that the default is 1%. Maybe something like multiplier for default # reps.

Reply:
Yeah, you're right. I'll change that.